god.js.org

Godow opened the pull request:
- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

Godow commented:
> Hi there!
> 
> The site you have linked to for your js.org domain request returns a 404: http://godow.github.io/
> Please fix this before we can process your js.org domain request.

Now its ok

indus commented:
I've added the subdomain to JS.ORGs zonefile for now (https://god.js.org). But I'm not sure what I shoudl think about the content. I really would have expected your personal page.
Could you explain to me how SDN is related to JavaScript?

Godow commented:
> I've added the subdomain to JS.ORGs zonefile for now (https://god.js.org). But I'm not sure what I shoudl think about the content. I really would have expected your personal page.
> Could you explain to me how SDN is related to JavaScript?

now I have change the content.

Godow commented:
> I've added the subdomain to JS.ORGs zonefile for now (https://god.js.org). But I'm not sure what I shoudl think about the content. I really would have expected your personal page.
> Could you explain to me how SDN is related to JavaScript?

now I have changed the content.

indus commented:
Welcome https://god.js.org

indus commented:
@Godow It seem like you page is no longer active. Please bring it back or remove the subdomain from the list.
